### ISSUE TITLE
mouse + keyboard combination remapping , especially when the mouse & keyboard is a different device

### ISSUE THREAD

**Issue by Maziar123** [sezanzeb/input-remapper#925]
Hi
Do these libs currently support mouse + keyboard combination remapping?

especially when the mouse & keyboard is a different device

**Comment by inakilbss** [comment]
i don't think so, from what i understand injector for each device are isolated from one another

**Comment by Maziar123** [comment]
But mix usable on the "user side," for example, CTRL + Right-click.
